# DM3 Chasis State Messages


###### CONSTANTS ######
uint8 MIN_POWER = 0
uint8 MAX_POWER = 100

###### MESSAGE ######

Header header

###################
# Chasis wheel power
###################

float32[] angle         # chasis angle
float32[] raw   	# pote meassure
